core/metadata.js

```diff
@@ -72,11 +72,35 @@ const CONFLICT_REGEXP = new RegExp(
   `-conflict-${DATE_REGEXP}${SEPARATOR_REGEXP}`
 )
 
+const LOCAL_ATTRIBUTES = [
```

Comment:
The  `name` is not kept here. What happens for this scenario?

1. A file `foo.txt` is synchronized
2. On the cozy, its content is updated
3. Cozy-desktop gets the changes feed
4. Before applying this change, cozy-desktop is stopped
5. The file is renamed locally
6. Cozy-desktop is restarted.

Reply:
And same question for the parent_id

Reply:
The `name` or rather `path` for local files, will be introduced later. For now, I don't need it to solve some problems and keeping it in sync is just more work.